assets/css/common.less

```diff
@@ -412,12 +412,8 @@
 			}
 
 			&[data-required=true] {
-				>div:first-child {
-					>div:only-child {
-						a {
-							display: none;
-						}
-					}
+				a[data-component="file-delete"] {
```

Comment:
It's better not to target JS handlers via CSS, these should be separate, this way we can rename handles knowing that only JS is affected. Maybe let's add some CSS class to these actions, based on the field container class via double underscore. Another temporary solution is just targeting the delete link directly without affecting other links.